datasets/hst.yaml

```diff
@@ -0,0 +1,26 @@
+Name: HST public data
+Description: |
+  The Hubble Space Telescope (HST) is one of the most productive scientific instruments ever created. This dataset contains calibrated and raw data for all of the currently active instruments on HST: ACS, COS, STIS and WFC3. 
+Documentation: http://astroquery.readthedocs.io/en/latest/mast/mast.html
+Contact: archive@stsci.edu
+UpdateFrequency: Hourly
+Tags:
+  - astronomy
+  - hst
+  - hubble
+  - nasa
+License: STScI herby grants the non-exclusive, royalty free, non-transferable, worldwide right and license to use, reproduce and publicly display in all media public data from the Hubble Space Telescope.
+Resources:
+  - Description: Project data files
+    ARN: arn:aws:s3:::stpubdata
+    Region: us-east-1
+    Type: S3 Bucket
+  - Description: Notifications for new data
```

Comment:
I don't think we have notifications set up on this bucket? Or perhaps we do and I just don't know? / cc @mfox22

Reply:
Some instructions on how to set this up at https://gist.github.com/jedsundwall/7b5ea0a33cc3ca0b9764f7090a59858a. Would be great to get this in place, if you need any help, just let me know.